src/BLSApkRegistry.sol

```diff
@@ -201,7 +185,22 @@ contract BLSApkRegistry is BLSApkRegistryStorage {
         uint256 index
     ) external view returns (bytes24) {
         ApkUpdate memory quorumApkUpdate = apkHistory[quorumNumber][index];
-        _validateApkHashAtBlockNumber(quorumApkUpdate, blockNumber);
+
+        /**
+         * Validate that the update is valid for the given blockNumber:
+         * - blockNumber should be >= the update block number
+         * - the next update block number should be either 0 or strictly greater than blockNumber
+         */
+        // TODO - should this fail if quorumApkUpdate.apkHash == 0? This will be the case for the first entry in each quorum
```

Comment:
it's fine? other checks in the sig checker will fail if no operators have registered for the quorum

can add if it feels safer tho